djproxy/views.py

```diff
@@ -135,6 +135,10 @@ def proxy(self):
         headers['X-Forwarded-Host'] = self.request.get_host()
         headers['X-Forwarded-For'] = self.xff
 
+        for prefix, url in self.reverse_urls:
+            if url == self.proxy_url and self.request.path.startswith(prefix):
```

Comment:
And that explains why my test (which went to /google/) was passing but
going to /sbbe/sites/ in yolacom failed. Thanks!

On Mon, Feb 2, 2015 at 4:13 AM, Thomas Welfley <notifications@github.com>
wrote:

> In djproxy/views.py
> <https://github.com/thomasw/djproxy/pull/13#discussion_r23913254>:
>
> > @@ -135,6 +135,10 @@ def proxy(self):
> >          headers['X-Forwarded-Host'] = self.request.get_host()
> >          headers['X-Forwarded-For'] = self.xff
> >
> > +        for prefix, url in self.reverse_urls:
> > +            if url == self.proxy_url and self.request.path.startswith(prefix):
>
> So, the behavior you'd see with this implemented as is:
>
> Going to /yahoo/ would be good. The prefix would be there. Going to
> /yahoo/yay/ would be broken, the prefix wouldn't be there.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thomasw/djproxy/pull/13/files#r23913254>.
>